render_delegate/light.cpp

```diff
@@ -400,9 +401,9 @@ class HdArnoldGenericLight : public HdLight {
     AtNode* _light;                    ///< Pointer to the Arnold Light.
     AtNode* _texture = nullptr;        ///< Pointer to the Arnold Texture Shader.
     AtNode* _filter = nullptr;         ///< Pointer to the Arnold Light filter for barndoor effects.
+    bool _supportsTexture = false;     ///< Value indicating texture support.
```

Comment:
Please change the constructor initialization order instead of changing the parameter order on the class. I generally order parameters by size on structs like this.